mava/evaluator.py

```diff
@@ -35,7 +35,7 @@ def get_ff_evaluator_fn(
     env: Environment,
     apply_fn: ActorApply,
     config: dict,
-    win_rate: bool = False,
+    include_win_rate: bool = False,
```

Comment:
I like being more explicit here, sorry one final thing: 
```suggestion
    log_win_rate: bool = False,
```